### PR TITLE
Style help and system pages

### PR DESCRIPTION
**Pull request by ciar4n**
### Before
image

### After
image

### CODE REVIEW

**Comment by wilsonge** [comment]
screen shot 2018-07-01 at 21 54 54

We have a php notice here - this is because in the definitions the icons are associated with the items not the headers. Needs a small tweak to get that working.

We need to think about how this looks on screens too - because even on my large pc screen (30 odd inches) - this is going off the edge of my screen. On my macbook screen you can only see the outer edge of the 4th column

screen shot 2018-07-01 at 21 55 57

Still a big improvement tho! Looking good!

File: administrator/components/com_cpanel/tmpl/help/default.php [hunk 1]
```diff
@@ -11,17 +11,14 @@
...
-			<div class="card-block">
-				<h4 class="card-title"><?php echo Text::_($link['label']); ?></h4>
-				<span class="fa fa-<?php echo $link['icon']; ?> fa-5x"></span>
-				<p class="card-text"><?php echo Text::_($link['desc']); ?></p>
-				<a href="<?php echo $link['link']; ?>" class="btn btn-primary"><?php echo Text::_($link['title']); ?></a>
-			</div>
-		</div>
-	<?php endforeach; ?>
+<div class="com-cpanel-help">
+	<h4 class="com-cpanel-help__header"><span class="fa fa-<?php echo $link['icon']; ?>"></span><?php echo Text::_('MOD_MENU_HELP'); ?></h4>
```
Comment (brianteeman) [inline administrator/components/com_cpanel/tmpl/help/default.php]: please add aria-hidden here and to other places in this PR with icons

**Comment by brianteeman** [comment]
4 columns is probably a bit adventurous

**Comment by dgrammatiko** [comment]
@brianteeman that was the design given

@ciar4n @wilsonge about the notification: I guess I included icons for each menu but not for the headers :(

**Comment by brianteeman** [comment]
@wilsonge i think its going off your screen just because of the error as without the error i cant replicate

Currently this isnt completely matching the issue #430 as the view is the same at all screen sizes

**Comment by wilsonge** [comment]
Yup - but in the designs none of the actual items have icons (aside from messages). So I guess it's best to do have arrays with

`name`, `icon`, `links`

**Comment by ciar4n** [comment]
For now I have removed the icons. Maybe someone with a little more smarts than I, could add them back in on another pr.

4 columns should be fine on most screen sizes. I suspect the overflow issue is related to the long strings in the php error. I added some media queries for the column count on smaller screens.

**Pull request merged**